Update info about changelog entry generation (#854)
packit/packit.dev: docs/fedora-releases-guide/index.md

you can specify the [`upstream_tag_template`](/docs/configuration/#upstream_tag_template)
configuration option so that Packit can extract the correct version.

> The changelog entry generated by default is `- Update to version <version>`.

Reviewer: Do we want to mention bug references?

If you use [`copy_upstream_release_description: true`](/docs/configuration/#copy_upstream_release_description),
the changelog entry will use the GitHub/GitLab release description field.
(Just make sure the formatting is compatible with spec file.
E.g. use `-` instead of `*` for lists to not create multiple changelog entries.)
There is also [`sync_changelog`](/docs/configuration/#sync_changelog) configuration option to enable syncing 
the whole changelog.
You can also utilize a [custom `changelog-entry` action](/docs/configuration/actions#syncing-the-release) if you would like to override the default behaviour.

        actions:
          changelog-entry:
            - bash -c 'echo "- New release ${PACKIT_PROJECT_VERSION}"'

> Besides the configuration of changelog via Packit, you can also use `%autochangelog` macro to
> generate the changelog entries automatically, see 
> [details](https://docs.pagure.org/Fedora-Infra.rpmautospec/autochangelog.html). In that case, Packit will not touch 
> the changelog at all.

Reviewer 2: ```suggestion
In case you prefer to use `%autochangelog` macro to generate the changelog entries automatically, see 
[details](https://docs.pagure.org/Fedora-Infra.rpmautospec/autochangelog.html), you can do that and Packit will not touch the changelog at all.
```

Reviewer 2: Ok, then can I suggest this (*besides* just doesn't feel right to me)?

```suggestion
Apart from updating changelog via Packit, you can also use `%autochangelog` macro to
generate the changelog entries automatically, see 
[details](https://docs.pagure.org/Fedora-Infra.rpmautospec/autochangelog.html). In that case, Packit will not touch 
the changelog at all.
```

You can override these URLs via [`sources`](/docs/configuration#sources) configuration key.
- You may utilise some of the [actions](/docs/configuration/actions/#syncing-the-release)
for overriding the Packit default behaviour, for example:
  - for the changelog entry generation, if you do not want the default `Update to version <version>`, you can use your own command(s):
  
        changelog-entry:

>           - bash -c 'git log --no-merges --pretty="format:- %s (%an)" $(git describe --tags --abbrev=0 ${PACKIT_PROJECT_UPSTREAM_TAG}^)..${PACKIT_PROJECT_UPSTREAM_TAG} --'

Reviewer 2: I would add a comment briefly explaining what this command does.
